test/features/install_dataindex.feature

```diff
@@ -89,4 +89,225 @@ Feature: Kogito Data Index
     }
     """
 
+#####
+
+  @failover
+  @events
+  @infinispan
+  @kafka
+  Scenario: Test Kogito Data Index failover with Infinispan
+    Given Infinispan Operator is deployed
+    And Infinispan instance "kogito-infinispan" is deployed with configuration:
+      | username | developer |
+      | password | mypass    |
+    And Install Infinispan Kogito Infra "infinispan" targeting service "kogito-infinispan" within 5 minutes
+
+    And Install Kogito Data Index with 1 replicas with configuration:
+      | config | infra | infinispan |
+      | config | infra | kafka      |
+    And Kogito Data Index has 1 pods running within 10 minutes
+
+    And Clone Kogito examples into local directory
+    And Local example service "process-quarkus-example" is built by Maven and deployed to runtime registry with Maven configuration:
+      | profile | persistence,events |
+    And Deploy quarkus example service "process-quarkus-example" from runtime registry with configuration:
+      | config | infra | infinispan |
+      | config | infra | kafka      |
+    And Kogito Runtime "process-quarkus-example" has 1 pods running within 10 minutes
+    
+    When Start "orders" process on service "process-quarkus-example" within 3 minutes with body:
+      """json
+      {
+        "approver" : "john",
+        "order" : {
+          "orderNumber" : "12345",
+          "shipped" : false
+        }
+      }
+      """
+    And Service "process-quarkus-example" contains 1 instances of process with name "orders"
+    And GraphQL request on Data Index service returns 1 instance of process with name "orders" within 2 minutes
+    And Scale Infinispan instance "kogito-infinispan" to 0 pods within 2 minutes
+    And GraphQL request on Data Index service returns 0 instances of process with name "orders" within 2 minutes
+    And Scale Infinispan instance "kogito-infinispan" to 1 pods within 2 minutes
+    And GraphQL request on Data Index service returns 1 instances of process with name "orders" within 2 minutes
+    And Start "orders" process on service "process-quarkus-example" within 3 minutes with body:
+      """json
+      {
+        "approver" : "john",
+        "order" : {
+          "orderNumber" : "12345",
+          "shipped" : false
+        }
+      }
+      """
+    
+    Then Service "process-quarkus-example" contains 1 instances of process with name "orders"
```

Comment:
So the service lost previous process? I would expect that it is still available as Infinispan use persistent volume.

Reply:
This is normal behavior as we use the cache service: https://infinispan.org/docs/infinispan-operator/main/operator.html#_data_grid_service
btw, we may need to change it to DataGrid (see the warning)

Reply:
Right, we should migrate soon.
Fine for me then.

Reply:
Created https://issues.redhat.com/browse/KOGITO-6237 and will try to do that next week